src/deploy/functions/services/index.ts

```diff
@@ -48,12 +49,12 @@ export const EVENT_SERVICE_MAPPING: Record<string, any> = {
 /**
  * Find the Service object for the given endpoint
  * @param endpoint the endpoint that we want the service for
- * @returns a Service object that corresponds to the event type of the endpoint or noop
+ * @return a Service object that corresponds to the event type of the endpoint or noop
  */
 export function serviceForEndpoint(endpoint: backend.Endpoint): Service {
   if (!backend.isEventTriggered(endpoint)) {
     return NoOpService;
   }
 
-  return EVENT_SERVICE_MAPPING[endpoint.eventTrigger.eventType] || NoOpService;
+  return EVENT_SERVICE_MAPPING[endpoint.eventTrigger.eventType as v2events.Event] || NoOpService;
```

Comment:
It's no less safe than previous iterations, but I kinda feel like if we're going to cast to a `v2events.Event` we should have a function that checks whether a string is a valid event type first.

Reply:
It isn't safer here, but it does important typos when we add new entries to the `EVENT_SERVICE_MAPPING`.

Agreed that having that check would've felt more idiomatic in hindsight.